### ISSUE TITLE
Version 3.7.0 is not working in node 6.14.2

### ISSUE THREAD

**Issue by fs-eire** [cmake-js/cmake-js#121]
```
> nvm install 6.14.2
> npm install -g cmake-js

> node -v
v6.14.2
> npm -v
3.10.10
> cmake-js
env: node\r: No such file or directory.
```

**Comment by unbornchikken** [comment]
I hate Git for Windows. Moment.

**Comment by unbornchikken** [comment]
I hope that I've fixed it as of 3.7.1. I don't have Linux/OSX by hand atm, so please test if it works at your side!

**Comment by fs-eire** [comment]
It looks good for 3.7.1 👍 

BTW, you may want to onboard a CI system (eg. [CircleCI](https://circleci.com/) [TravisCI](https://travis-ci.org/)) to build, test and publish npm packages automatically to get rid of potential future similar issues.
